R/read.R

```diff
@@ -0,0 +1,127 @@
+#' Convert an Athena type to an Arrow type
+#'
+#' @param t A string giving an Athena type
+#'
+#' @return An Arrow type
+#' @export
+#'
+#' @see https://docs.aws.amazon.com/athena/latest/ug/data-types.html
+#' @see https://arrow.apache.org/docs/r/reference/data-type.html
+convert_athena_type_to_arrow <- function(t) {
```

Comment:
looks like this function isn't used in the currently-implemented `read_sql_query` function. I think for these vestigial bits, it's probably worth removing and shifting to a develop branch.

Reply:
[2ae0bf5](https://github.com/moj-analytical-services/dbtools/pull/30/commits/2ae0bf565e446eaba537c2284c3eea09b846a725)